src/components/Modal/Modal.vue

```diff
@@ -728,20 +733,22 @@ $header-size: 50px;
 	&--small {
 		.modal-container {
 			width: 350px;
+			max-height: 90%;
 			height: 450px;
 		}
 	}
 	&--medium {
 		.modal-container {
 			width: 500px;
+			max-height: 90%;
```

Comment:
having a max-heigt is necessary e.g. if you rotate a smartphone that is only 350px wide into landscape mode (which would then be 350px high)